src/hooks/useUnipoolRewards.ts

```diff
@@ -31,7 +30,7 @@ export default function useUnipoolRewards() {
         console.error(`Error fetching earned rewards: ${err}`)
       }
 
-      timer = setTimeout(fetchEarned, 5000)
+      timer = window.setTimeout(fetchEarned, 5000)
```

Comment:
Consider using constant to the time 5000

Reply:
Since it was used in just one place, it doesn't make sense to me having a constant.